lib/scrape.js

```diff
@@ -241,6 +285,10 @@ async function fetchOrgsWithData() {
         platform: chattie.CHAT[orgChats[index].type],
         image: CHAT_IMAGES[chattie.CHAT[orgChats[index].type]],
       },
+      wiki: {
+        url: orgWiki[index].url,
+        type: orgWiki[index].type,
+      }
```

Comment:
Add trailing comma

Reply:
Was in the process of linting, done now.